airflow/providers/google/cloud/hooks/cloud_build.py

```diff
@@ -123,9 +123,13 @@ def cancel_build(
 
         self.log.info("Start cancelling build: %s.", id_)
 
-        build = client.cancel_build(
-            request={'project_id': project_id, 'id': id_}, retry=retry, timeout=timeout, metadata=metadata
+        cancel_build = client.cancel_build(
+            request={'project_id': project_id, 'id': id_},
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
         )
+        build = cancel_build
```

Comment:
What was MyPy complaining about here to update the variable name from `build` to `cancel_build`?  Would it make sense to return cancel_build` rather than set `build = cancel_build`?

Reply:
Good point let me try. Might be some earlier attempt trickle to final change.

Reply:
Yep. Reverted it.